examples/blog-starter/posts/index.js

```diff
@@ -0,0 +1,7 @@
+import preval from 'babel-plugin-preval/macro'
```

Comment:
Is this file required for something ? `get-blog-posts` will be handled in build time anyway